certif/app/templates/components/routes/authenticated/sessions/list-items.hbs

```diff
@@ -1,12 +1,40 @@
-<div class="list-sessions-page__header-page">
+<div class="session-list__header">
   <h1 class="page__title page-title">Sessions de certification</h1>
-  <div class="list-sessions-page__create-session-button">
-    {{link-to 'Créer une session +' 'authenticated.sessions.list' class="button button--link"}}
-  </div>
 </div>
 
-<div class="session-list">
-  {{#each sortedSessions as |session|}}
-    {{routes/authenticated/sessions/session-item session=session}}
-  {{/each}}
+<div class="session-list panel panel--white">
+  <div class="session-list__table-header">
+    {{#link-to 'authenticated.sessions.list' class="button button--link button--with-icon"}}
+      Créer une session
+      {{fa-icon 'plus'}}
+    {{/link-to}}
+  </div>
+
+  {{#ember-table class="session-list__table" as |table|}}
+    {{#table.head class="session-list__table-head" columns=columns as |head|}}
+      {{#head.row class="session-list__table-head-row" as |row|}}
```

Comment:
On défini des classes css ici, qu'on ne retrouve pas dans les fichiers css ... why that ?

Reply:
même raison : "à la base j'avais créé des classes, mais @jbuget préfère utiliser les éléments de base pour pouvoir les réutiliser sur un autre tableau". Donc soit on garde les éléments de base et on supprime les classes dans le `.hbs`, soit on fait tout par classes. Qu'en penses-tu ?